### DISCUSSION TITLE
Filtering out keyword in title

### DISCUSSION THREAD

**Discussion by ghost** [trix7777/m2strm#21] [Ideas]
Does your code have a way to filter out words usually at the beginning of the title name. For example, French - Title of the Show, French - Title of Show2, etc. I could filter it with sed in linux just wondering if you have the code built in.

**Comment by trix7777** [comment]
Not as it is coded right now, no. But I will take it into consideration next time I have time to make an upgrade. Thanks for the suggestion!
